packages/terra-clinical-item-collection/src/Item.jsx

```diff
@@ -183,4 +203,13 @@ Item.defaultProps = defaultProps;
 Item.Display = ItemView.Display;
 Item.Comment = ItemView.Comment;
 
+const opts = {
+  Layouts: layouts,
+  EmphasisTypes: emphasisTypes,
+  Alignments: alignments,
+  Views: views,
```

Comment:
I am thinking these enum's names should be matching the name of the prop they are correlated with. For example: `Alignments ` to `accessoryAlignments`

Reply:
This was done in https://github.com/cerner/terra-clinical/pull/510/commits/3bbac786827c62570528d93a7eae6ac15faf5314